### ISSUE TITLE
Controller buttons dont always appear with youtube videos

### ISSUE THREAD

**Issue by DesperatePea** [tnychn/media-controller-extension#5]
Sometimes the controls buttons dont appear when listening to youtube videos. 
![image](https://github.com/tnychn/media-controller-extension/assets/73502428/2a529b4a-946a-4945-bcfe-10716de2f920)

Firefox 123.0 (64-bit)
Mac Os Sonoma

**Comment by tnychn** [comment]
This is a known issue. The controls are hidden if the extension cannot find the corresponding media element from DOM.
Possibly due to the tab having been discarded and reactivated before.

**Comment by ygoe** [comment]
Yes, I also had this now. Just installed the add-on to see how it works. I was puzzled about that (/) button and observed strange behaviour after clicking it. I had to close the tab and reopen, then reload it again to get YouTube playback controls back.

In the source code I finally found out that it unloads the tab without closing it. A tooltip on the buttons would have been nice.

**Comment by trlkly** [comment]
I notice this on other sites, too. I suspect it is because they use iframes. Finding the video (or audio) element that is playing is a bit more cumbersome when you need to deal with iframes. You have to have it where the injected code runs in the iframe, but then communicates back with the extension.

**Comment by vlakoff** [comment]
I might suggest to add a bit of spacing between the "Remove from List" and "Close Tab" buttons, to prevent accidentally clicking on the former instead of the latter.

Rationale: accidentally clicking on "Remove from List" may be frustrating, as in order to get back the relevant tab in the extension popup, the user has to find back the browser tab and "reactivate" it by some means (e.g. reload page, pause-play having sound enabled…)

Similarly, I suppose that accidentally clicking on the latter instead of the former may be frustrating too.

**Comment by vlakoff** [comment]
Posted on the wrong issue, please ignore previous notification.
